### ISSUE TITLE
log10 or log in LogParabola definition?

### ISSUE THREAD

**Issue by cosimoNigro** [gammapy/gammapy#1155]
Hi all,

when the LogParabola function is evaluated the np.log function is used 
```python
    @staticmethod
    def evaluate(energy, amplitude, reference, alpha, beta):
        """Evaluate the model (static function)."""
        # TODO: can this comment be removed?
        # cast dimensionless values as np.array, because of bug in Astropy < v1.2
        # https://github.com/astropy/astropy/issues/4764
        try:
            xx = (energy / reference).to('')
exponent = -alpha - beta * np.log(xx)
```
shouldn't  **np.log10** be used? 
I was checking the recently added MAGIC Crab spectrum and I realized that, though the parametrization is correct, I could not obtain the same spectrum. In MAGIC paper and software the log10 is used.
Also in sherpa the log parabola is defined as log10
http://cxc.harvard.edu/sherpa/ahelp/logparabola.html
in the Fermi model definition there is only _log_ indicated but I think it means log10 also.
https://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/source_models.html
What do you think? 
```python
import numpy as np
import matplotlib.pyplot as plt
import astropy.units as u

E = np.logspace(np.log10(0.05), np.log10(20), 100) * u.TeV 

amplitude = 3.23 * 1e-11 * u.Unit('TeV-1 cm-2 s-1')
alpha = 2.47
beta = 0.24 
E0 = 1 * u.TeV

def LP_log10(E):
        return amplitude * (E/E0)**(-alpha -beta*np.log10(E/E0))

def LP_log(E):
        return amplitude * (E/E0)**(-alpha -beta*np.log(E/E0))

plt.loglog(E, LP_log10(E), lw=2, label='LogParabola np.log10')
plt.loglog(E, LP_log(E), lw=2, label='LogParabola np.log')

plt.legend()
plt.show()
```
![figure_1](https://user-images.githubusercontent.com/26409711/30866074-8fe57c22-a2d8-11e7-8955-bb0dcdd8c28c.png)

**Comment by joleroi** [comment]
Thanks for this issue, I remember loosing a day on this some time ago.

FYI: 
* In the [H.E.S.S. software](https://bitbucket.org/hess_software/flux/src/e67e2a24042036dc3d8d9598cbe2465ffe56320a/src/FitFunction.C?at=master&fileviewer=file-view-default#FitFunction.C-2085) the natural logarithms is used.

* In [this paper](http://cdsads.u-strasbg.fr/cgi-bin/bib_query?2000A%26A...361..695M) they talk about log-parabola models for the Crab but as far as I can see, they never say whether they mean the natural logarithm or log10.

**Comment by woodmd** [comment]
I can add that the Fermi software uses natural log not log10.

**Comment by cosimoNigro** [comment]
Thank you @joleroi, @woodmd
IMHO the _Log_ in the paper is also log10, because is in capital letter in the notation 
https://en.wikipedia.org/wiki/Common_logarithm
it is what I remember from school at least.
As the paper says the purpose of the log-parabola is to have a Power-Law constantly increasing in slope with the energy (and the axis values increase with log10).
Let me know what do you think, otherwise I will modify the MAGIC Crab Spectrum.

Thank you

**Comment by joleroi** [comment]
I checked again, sherpa definitely uses [log10](https://github.com/sherpa/sherpa/blob/34b2f05d0bbe892c01c67badb1031b5dbf649acd/sherpa/include/sherpa/models.hh#L672)

Since both seem to be used 'in the wild' I guess we can go either way, as long as we state it in the docs. I slightly favor ``log10`` for the arguments that @cosimoNigro has presented. This however would require an (for the user potentially annoying) change in gammapy, so maybe I do prefer to leave things as they are. I offer do to a PR summarizing this issue in the docstring of ``LogParabola``

**Comment by cdeil** [comment]
Please keep natural log.

I think unless there's a strong reason (which MAGIC / HESS software isn't) we should just implement the parametrisations from Gammalib / ctools, which are a cleaned-up evolution of the Fermi ST (see #669 for further info). I would like to work on modeling for Gammapy, and started a little bit [here](https://github.com/gammapy/gamma-cat/tree/master/model_serialisation), but it doesn't look like I'll have time to do it.

A PR to clarify the docstring would be useful (maybe adjust the Crab reference spec if needed, maybe add a helper function to convert between the two parametrisations if you think it it useful).

**Comment by cdeil** [comment]
Fixed by @joleroi  in #1159 .
